src/test/java/space/obminyashka/items_exchange/service/AdvertisementServiceIntegrationTest.java

```diff
@@ -34,4 +35,22 @@ void findRandom12Thumbnails_shouldReturnEmpty() {
         final var secondTitlesGetAttempt = advertisementService.findRandom12Thumbnails();
         assertEquals(firstTitlesGetAttempt, secondTitlesGetAttempt, "Collections must be equals because of caching response");
     }
+
+    @Test
+    @Transactional
+    @DataSet("database_init.yml")
+    void findPaginated_shouldReturnEmpty() {
```

Comment:
Add @ExpectedDataSet to compare data

Reply:
I don't think it's possible to compare it with ExpectedDataSet, because the request wouldn't change the database's content.